### DISCUSSION TITLE
Unable to load extension

### DISCUSSION THREAD

**Discussion by yoonghm** [duckdb/duckdb#7925] [Q&A]
Hi,

I have installed the extension `spatial` and try to load it. It gave the following error:

```
Error: IO Error: Extension "C:\Users\username\.duckdb\extensions\v0.8.0\windows_amd64\spatial.duckdb_extension" could not be loaded: Access is denied.
```

My company has installed a software to prevent us from running software from unauthorized directories. I copy the extension file `spatial_duckdb_extension` to `D:\anaconda3`, which I could run or load any program or library.

From a duckdb shell, I tried this

```
set search_path='D:\\anaconda3'
```

or this

```
set search_path='D:\anaconda3'
```

It gave the following error:

```
Error: Binder Error: Catalog "D:\\anaconda3" does not exist!
```

or

```
Error: Binder Error: Catalog "D:\anaconda3" does not exist!
```

Do you have suggestions?

**Comment by hannes** [comment]
Try 

```SQL
SET extension_directory='D:\anaconda3';
```

**Comment by yoonghm** [comment]
Yes, it tried to load from `D:\anaconda3`. However `spatial` is not designed for Windows OS.

![image](https://github.com/duckdb/duckdb/assets/4215991/95b8b280-cf7c-40e8-9723-7215b7e1a52d)

**Reply by Mause** [reply]
You shouldn't be getting that error, did you install spatial using DuckDB or manually?

**Reply by yoonghm** [reply]
I installed it via duckdb CLI. Perhaps the execution was blocked by my company. Let me try on a personal laptop and revert.

**Comment by yoonghm** [comment]
Yes. It worked. The problem was caused by whitelisting software preventing the software from running.
